subsys/bluetooth/mesh/cfg_cli.c

```diff
@@ -141,7 +141,7 @@ static int krp_status(struct bt_mesh_model *model, struct bt_mesh_msg_ctx *ctx,
 	       bt_hex(buf->data, buf->len));
 
 	if (!bt_mesh_msg_ack_ctx_match(&cli->ack_ctx, OP_KRP_STATUS, ctx->addr,
-				       (void **)&param)) {
+				       (void **)&param) || !param) {
```

Comment:
Actually, it's not possible for param to be NULL in any of these. I believe Coverity fails because we were doing a NULL check after accessing the variable, which would either be useless (if param could never be null) or invalid (if it could be null). Since it's useless, I'll just remove the checks altogether instead.